firebase_admin/_auth_client.py

```diff
@@ -36,17 +39,42 @@ def __init__(self, app, tenant_id=None):
             2. set the project ID explicitly via Firebase App options, or
             3. set the project ID via the GOOGLE_CLOUD_PROJECT environment variable.""")
 
-        credential = app.credential.get_credential()
+        credential = None
         version_header = 'Python/Admin/{0}'.format(firebase_admin.__version__)
+        http_headers = {'X-Client-Version': version_header}
+        # None lets us fallback to default endpoint URLs if an emulator isn't present.
+        id_toolkit_endpoints = {
+            "v1": None,
+            "v2beta1": None,
+        }
+
+        # If an emulator is present, check that the given value matches the expected format and set
+        # endpoint URLs to use the emulator. Also set a fake authorization token.
+        emulator_host = os.environ.get(_EMULATOR_HOST_ENV_VAR)
+        if emulator_host:
+            if '//' in emulator_host:
+                raise ValueError(
+                    'Invalid {0}: "{1}". It must follow format "host:port".'.format(
+                        _EMULATOR_HOST_ENV_VAR, emulator_host))
+            # Used instead of credential
+            http_headers["Authorization"] = "Bearer owner"
```

Comment:
This turned out to be a bit tricky - one of tests (`tests/test_user_mgt.py::TestImportUsers::test_import_users_http_error`) would fail trying to refresh the token. So I have settled in favour of using credentials if provided, or falling back to setting the token if not.